client/src/components/event.test.tsx

```diff
@@ -0,0 +1,28 @@
+// src/components/Event.test.tsx
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import Event, { type EventProps } from "./event";
+
+describe("Event component", () => {
+    const baseProps: EventProps = {
+        id: "1",
+        title: "Testovací událost",
+        location: "Praha",
+        dates: [
+            {
+                timestamp: Date.now(),
+                records: [{ name: "Karel", answer: "yes" }],
+            },
+        ],
+    };
+
+    it("zobrazí název události", () => {
```

Comment:
jestli dobre koukam, tahle komponenta `event` neni prece nikde pouzita? :) Pouzivas EventDetail, cili testama jsi pokryla nepouzity kod :) A tim padem nemas otestovano, kdyz napr. to REST API vrati chybovy kod